nixos/modules/programs/nix-monitored.nix

```
@@ -0,0 +1,40 @@
+{ pkgs, lib, config, ... }:
+
+let
+  cfg = config.nix.monitored;
+in
+{
+  meta.maintainers = [ lib.maintainers.ners ];
+
+  options.nix.monitored = with lib; {
+    enable = mkEnableOption (mdDoc "nix-monitored, an improved output formatter for Nix");
```

```suggestion
    enable = mkEnableOption "nix-monitored, an improved output formatter for Nix";
```

etc. mdDoc should be removed.